Add mapping decisions for identifiers.

mlhale7 opened the pull request:
**Jira Issue**: [DIGITAL-585](https://jirautk.atlassian.net/browse/DIGITAL-585)

## What does this Pull Request do?

This PR adds mapping decisions and justifications for the following Xpaths associated with the top-level element identifier:

mods:identifer[@type="Vendor ID"]
mods:identifier[@type="archival number"]
mods:identifier[@type="catalog"]
mods:identifier[@type="circular"]
mods:identifier[@type="cw"]
mods:identifier[@type="document ID"]
mods:identifier[@type="documentID"]
mods:identifier[@type="filename"]
mods:identifier[@type="film number"]
mods:identifier[@type="legacy"]
mods:identifier[@type="local"]
mods:identifier[@type="original ID"]
mods:identifier[@type="photograph number"]
mods:identifier[@type="slide number"]
mods:identifier[@type=”pid”]
mods:identifier[@type=”opac”]
mods:identifier[@type="spc"]
mods:identifier[@type="acquisition"]
mods:identifier[@type="oclc"]
mods:identifier[@type="issn"]
mods:identifier[@type="isbn"]

## How should this be tested?

Review the mapping doc for any syntax or content errors. Also make sure that no identifiers from our [analysis doc](https://utk-mods-to-rdf.readthedocs.io/en/latest/top_level_elements/identifier.html) have been left unaddressed besides mods:identifier[@type="Work ID"]. I'll note that we need to discuss this at our next meeting as it was inadvertently left off.

## Additional Notes:

Writing this up, I realized that we use DBpedia for most things, but opted to use Standard Identifier Schemes for our local identifiers.

## Interested parties

@markpbaggett @CanOfBees @mathewjordan

markpbaggett commented on docs/source/contents/4_draft_mapping.rst:
```diff
@@ -65,6 +65,246 @@ Mapping
+Acquisition Identifier
+----------------------
+
+Use Case
+
+^^^^^^^^
```
This heading is bad:

```
Use case
--------

```

markpbaggett commented on docs/source/contents/4_draft_mapping.rst:
```diff
@@ -531,9 +771,25 @@ typeOfResource
 classification
 ==============
 
-relateItem
+relatedItem
 ==========
```
When you fixed the spelling, you didn't update the heading to be the same length.  This needs to be fixed.

markpbaggett commented on docs/source/contents/4_draft_mapping.rst:
```diff
@@ -531,9 +771,25 @@ typeOfResource
+Justification
+^^^^^^^^^^^^^
+
+Xpath
+^^^^^
```
Add a newline after the header.

markpbaggett commented on docs/source/contents/4_draft_mapping.rst:
```diff
@@ -531,9 +771,25 @@ typeOfResource
+Xpath
+^^^^^
+mods: relatedItem[@type="otherVersion"]/mods:identifier[@type="catalog"]
+Decision
+^^^^^^^^
```
Add newlines above and below.

mlhale7 commented:
@markpbaggett - I believe my most recent commit has addressed these issues. Let me know if I missed anything and thank you so much for closely reviewing.
